### ISSUE TITLE
Extend LoadDNSSCD to load raw data

### ISSUE THREAD

**Issue by mganeva** [mantidproject/mantid#23203]
Request of the DNS instrument scientist.

**Before starting to implement this, ensure, that PR #23029 is merged**

`LoadDNSSCD` should be extended to be able to load the raw data in `2theta, omega, tof` axes. New property `Axes` should allow user to choose in the drop-down menu whether the workspace should contain `H, K, L, dE` or `2theta, omega, tof`.  

In this notation:
`2theta` --- is a scattering angle in degrees. It is calculated as (see line `553` in `LoadDNSSCD.cpp`)
```c++
// det_id is the detector ID, a number from 0 to 23
// deterota is the angle of the detector bank rotation, it is given in the data file
// 5.0 degrees is a gap between detectors
double two_theta = det_id*5.0 - deterota
```

`omega` --- is a sample rotation angle in degrees. It is calculated as
```c++
// huber is an angle given in the data file
// deterota is given in the data file
double omega = huber - deterota
```

`tof` is a total neutron time-of-flight  and should be in the microseconds. It is calculated as
```c++
// time-of-fligt from chopper to sample, see line 449
// L1 --- is a distance from chopper to sample, velocity is a neutron velocity, see line 447
double tof1 = 1.0e+06*L1/velocity
```
```c++
// for elastic data (with the only one time channel), see line 539
// L2 is the distance from sample to detector
double tof2_elastic = 1.0e+06*L2/velocity
double tof = tof1 + tof2_elastic
```
```c++
// for TOF data (more than 1 time channel), see line 561
// channel_number is a number from 0 to number of time channels
// channel_width is the with of the time channel in microseconds; given in the data file
double tof2 = channel_number*channel_width + 0.5*channel_width
double tof = tof1 + tof2
```
If `ElasticChannel` property is given, the TOF data array should be rotated to get elastic peak at the correct position. See lines `541-551`.

This feature would  generalize algorithm and make it applicable also for powder data. After this will be implemented, it might make sence to rename `LoadDNSSCD` to `LoadDNSMD` (will be a separate issue).

I will assign myself, but @JoachimCoenen may also work on it.
